network-libp2p/src/discovery/peer_contacts.rs

```diff
@@ -61,117 +28,44 @@ bitflags! {
         ///
         const FULL_BLOCKS = 1 << 0;
 
-        /// The node provides the full block history.
-        ///
-        /// If {@link Services.FULL_BLOCKS} is set, these blocks are provided as full blocks.
-        ///
-        const BLOCK_HISTORY = 1 << 1;
-
-        /// The node provides a proof that a certain block is included in the current chain.
-        ///
-        /// If [[`Services::FULL_BLOCKS`] is set, these blocks may be requested as full blocks.
+        /// The node provides the full transaction history.
         ///
-        /// However, if [`Services::BLOCK_HISTORY`] is not set, this service is only provided for the latest
-        /// [`nimiq_primitives::policy::NUM_BLOCKS_VERIFICATION`] blocks.
-        ///
-        const BLOCK_PROOF = 1 << 2;
+        const HISTORY = 1 << 1;
 
-        /// The node provides a chain proof for the tip of the current main chain.
+        /// The ZKP that proves the latest election block
```

Comment:
```suggestion
        /// The node provides the ZKP for the latest election block
```
(proof is duplicated since `ZKP` already includes proof)